src/features/status-bar/components/help-menu-items.jsx

```
@@ -0,0 +1,10 @@
+import { isElectron, openExternal } from 'utils/electron';
+
+export const HelpMenuItemsLabel = ({ url, name }) => {
+  const navigate = () => {
+    if (isElectron()) openExternal(url);
+    else window.open(url, '_blank', 'noopener,noreferrer');
+  };
+
+  return <div onClick={navigate}>{name}</div>;
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)